src/api/response.ts

```diff
@@ -4,6 +4,7 @@ import { TuyaDeviceDefaults } from "../config";
 export type ExtendedBoolean = boolean | "true" | "false" | "True" | "False";
 
 type TuyaProperties = Partial<{
+  max_brightness: number;
```

Comment:
Please move this to the `CustomProperties` below. 
The `TuyaProperties` is used for values that might be provided by the Tuya API which this property isn't.

Reply:
Done